### PR TITLE
Migrate asp.net code from common

### PR DESCRIPTION
**Pull request by bradenedmunds**

### CODE REVIEW

File: src/Cortside.AspNetCore.Tests/MessageExceptionResponseFilterTest.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+            // assert
+            Assert.True(context.ExceptionHandled);
+            Assert.NotNull(context.Result);
+            Assert.True(comparison(context.Result));
+        }
+
+        [Fact]
+        public void ShouldGenerateErrorModel() {
+            var filter = new MessageExceptionResponseFilter(new Logger<MessageExceptionResponseFilter>(loggerFactory));
```
[the remark this answers is not in the document]
Comment (bradenedmunds) [reply src/Cortside.AspNetCore.Tests/MessageExceptionResponseFilterTest.cs]: Fixed.

File: src/Cortside.AspNetCore.Tests/MessageExceptionResponseFilterTest.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+            // assert
+            Assert.True(context.ExceptionHandled);
+            Assert.NotNull(context.Result);
+            Assert.True(comparison(context.Result));
+        }
+
+        [Fact]
+        public void ShouldGenerateErrorModel() {
+            var filter = new MessageExceptionResponseFilter(new Logger<MessageExceptionResponseFilter>(new LoggerFactory()));
```
[the remark this answers is not in the document]
Comment (bradenedmunds) [reply src/Cortside.AspNetCore.Tests/MessageExceptionResponseFilterTest.cs]: Fixed.

[the remark this answers is not in the document]
Comment (bradenedmunds) [reply src/Cortside.AspNetCore.Tests/MessageExceptionResponseFilterTest.cs]: Fixed.

File: src/Cortside.AspNetCore.Tests/MessageExceptionResponseFilterTest.cs [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+using Microsoft.Extensions.Logging;
+using Moq;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Cortside.Common.Messages.Tests.Filters {
+    public class MessageExceptionResponseFilterTest {
+        private readonly MessageExceptionResponseFilter filter = null;
+        public MessageExceptionResponseFilterTest() {
+            filter = new MessageExceptionResponseFilter(new Logger<MessageExceptionResponseFilter>(new LoggerFactory()));
```
[the remark this answers is not in the document]
Comment (bradenedmunds) [reply src/Cortside.AspNetCore.Tests/MessageExceptionResponseFilterTest.cs]: Fixed.

File: src/Cortside.AspNetCore.Tests/Middleware/CorrelationIdTest.cs [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        public async Task ReturnsRequestCorrelationIdInResponseHeader() {
+            var builder = new WebHostBuilder()
+               .Configure(app => app.UseMiddleware<CorrelationMiddleware>())
+               .ConfigureServices(services => {
+                   services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+               });
+
+            using var server = new TestServer(builder);
+
+            var request = new HttpRequestMessage();
```
[the remark this answers is not in the document]
Comment (bradenedmunds) [reply src/Cortside.AspNetCore.Tests/Middleware/CorrelationIdTest.cs:47]: Fixed.

File: src/Cortside.AspNetCore/Filters/Results/ValidationFailedResult.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+using Cortside.Common.Messages.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Cortside.AspNetCore.Filters.Results {
+    public class ValidationFailedResult : ModelStateResult {
+        public ValidationFailedResult(ModelStateDictionary modelState) : base(modelState) {
+            var errors = modelState.Keys
+                        .SelectMany(key => modelState[key].Errors.Select(x => new ErrorModel(x.Exception?.GetType()?.Name ?? "ModelStateValidation", key, x.ErrorMessage)))
+                        .ToList();
```
[the remark this answers is not in the document]
Comment (bradenedmunds) [reply src/Cortside.AspNetCore/Filters/Results/ValidationFailedResult.cs]: Removed code.

**Review by cYCL157 — APPROVED** [review]

**Pull request merged**